OrbitQt/servicedeploymanager.h

```diff
@@ -26,7 +26,6 @@ class ServiceDeployManager : public QObject {
 
  public:
   struct Ports {
```

Comment:
Maybe we should rename that struct to "GrpcPort" and use it as a strong typedef.

Reply:
Agree, done.